### PR TITLE
[zh] Sync ambient/usage/l4-policy/index.md

### PR DESCRIPTION
**Pull request by windsonsea**
Sync with en text

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/ambient/usage/l4-policy/index.md [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
+description: 仅使用 L4 安全覆盖时支持的安全特性。
 weight: 20
 owner: istio/wg-networking-maintainers
 test: no
 ---
 
-在 Istio Ambient 模式中，{{< gloss >}}ztunnel{{< /gloss >}} 和
-{{< gloss >}}waypoint{{< /gloss >}} 代理采用分层机制，
-让您可以选择是否希望为给定的工作负载启用 Layer 7（L7）进行处理。
+Istio [安全策略](/zh/docs/concepts/security) 的四层（L4）特性由
```
Comment (wilsonwu) [inline content/zh/docs/ambient/usage/l4-policy/index.md:9]:
```suggestion
Istio [安全策略](/zh/docs/concepts/security)的四层（L4）特性由
```

File: content/zh/docs/ambient/usage/l4-policy/index.md [hunk 2]
```diff
@@ -1,28 +1,25 @@
...
-要使用 L7 策略和 Istio 的流量路由功能，
-您可以为您的工作负载[部署一个 waypoint](/zh/docs/ambient/usage/waypoint)。
+## 使用 ztunnel 强制执行策略 {#policy-enforcement-using-ztunnel}
 
-## Layer 4 鉴权策略 {#layer-4-authorization-policies}
-
-当工作负载在安全覆盖模式下注册时，ztunnel 代理会实施鉴权策略。
-
-实际的执行点位于连接路径中的接收（服务端）ztunnel 代理的位置。
+当某个工作负载注册到 {{< gloss "Secure L4 Overlay" >}}安全覆盖模式{{< /gloss >}}时，
```
Comment (wilsonwu) [inline content/zh/docs/ambient/usage/l4-policy/index.md:21]:
```suggestion
当某个工作负载注册到{{< gloss "Secure L4 Overlay" >}}安全覆盖模式{{< /gloss >}}时，
```

File: content/zh/docs/ambient/usage/l4-policy/index.md [hunk 3]
```diff
@@ -82,23 +105,34 @@ spec:
...
+当将 waypoint 代理添加到工作负载时，您现在有两个地方可以强制执行 L4 策略。
+（L7 策略只能在 waypoint 代理处执行。）
 
-由于 ztunnel 和 {{< gloss >}}HBONE{{< /gloss >}} 中隐式使用了 mTLS，
-因此无法在策略中使用 `DISABLE` 模式。此类策略将被忽视。
+仅使用安全覆盖时，流量会在目标 ztunnel 处以**源**工作负载的身份出现。
 
-如果您需要禁用整个命名空间的 mTLS，则必须禁用 Ambient 模式：
+waypoint 代理不会伪装源工作负载的身份。一旦您将 waypoint 引入流量路径，
+目标 ztunnel 将看到带有**waypoint**身份的流量，而不是源身份。
```
Comment (wilsonwu) [inline content/zh/docs/ambient/usage/l4-policy/index.md:122]:
```suggestion
目标 ztunnel 将看到带有 **waypoint** 身份的流量，而不是源身份。
```

File: content/zh/docs/reference/glossary/secure-l4-overlay.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 安全 L4 覆盖
+title: Secure L4 Overlay
```
Comment (windsonsea) [inline content/zh/docs/reference/glossary/secure-l4-overlay.md:2]: If this title is translate, `{{< gloss "Secure L4 Overlay" >}}安全覆盖模式{{< /gloss >}}` will fail.

Comment (wilsonwu) [reply content/zh/docs/reference/glossary/secure-l4-overlay.md:2]: So, is that mean all gloss title in zh content should not be translated?

Comment (windsonsea) [reply content/zh/docs/reference/glossary/secure-l4-overlay.md:2]: Yes, currently. Only `glossary/index.md`has a Chinese title.

Comment (windsonsea) [reply content/zh/docs/reference/glossary/secure-l4-overlay.md:2]: Only a few were translated.

**Review by wilsonwu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**